src/routes/AddressBook.tsx

```diff
@@ -0,0 +1,208 @@
+import { FC } from 'react'
+import {
+  Box,
+  Button,
+  CommonTable,
+  Flex,
+  HexFish,
+  Icon,
+  IconButton,
+  IconSearch,
+  NAMED_COLORS,
+  SearchAutocomplete,
+  SelectField,
+  chakra,
+} from '@ironfish/ui-kit'
+import IconAdd from '@ironfish/ui-kit/dist/svgx/icon-add'
+import Send from 'Svgx/send'
+import Caret from 'Svgx/caret-icon'
+
+const getIconBg = (address = '') => {
+  let colorNumber = 0
+  Array.from(address).forEach(char => {
+    colorNumber += char.charCodeAt(0)
+  })
+
+  return `hsl(${colorNumber % 255}, 100%, 73%)`
+}
+
+const DEMO_DATA = [
+  {
+    name: 'Frankie Boy',
+    address: '789wenft893ntw5v780ntq304wnv5t370q8nt5q340',
+  },
+  {
+    name: 'Tweetie',
+    address: '789wenft893ntw5v780ntq304wnv5t370q8nt5q340',
+  },
+  {
+    name: 'Rox1923',
+    address: '789wenft893ntw5v780ntq304wnv5t370q8nt5q340',
+  },
+  {
+    name: 'Alfred A',
+    address: '789wenft893ntw5v780ntq304wnv5t370q8nt5q340',
+  },
+  {
+    name: 'Derek',
+    address: '789wenft893ntw5v780ntq304wnv5t370q8nt5q340',
+  },
+  {
+    name: 'Jason',
+    address: '789wenft893ntw5v780ntq304wnv5t370q8nt5q340',
+  },
+]
+
+const AddressBook: FC = () => (
+  <Flex
+    width="100%"
+    height="100%"
+    justifyContent="center"
+    alignItems="start"
+    px="2rem"
+    py="2.5rem"
+  >
+    <Box width="100%" height="100%" maxWidth="65.5rem">
+      <Flex
+        mb="2.5rem"
+        justifyContent="space-between"
+        w="100%"
+        alignItems="center"
+      >
+        <Flex direction="column">
+          <Box>
+            <h2>Address Book</h2>
+          </Box>
+        </Flex>
+        <Flex>
+          <Button
+            leftIcon={<IconAdd />}
+            borderRadius="4rem"
+            variant="secondary"
+          >
+            Import Account
+          </Button>
+        </Flex>
+      </Flex>
+      <Flex justifyContent="space-between" w="100%" alignItems="center">
+        <SearchAutocomplete<string>
```

Comment:
Do we need autocomplete here for me it looks like it should update results in the table below?

Reply:
Fixed.